classes/game.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+require 'date'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    super && Date.today.year - Date.parse(@publish_date).year > 2
+  end
```

Comment:
In the can_be_archived? method, it should return TRUE if the parent's method returns true and if `last_played_at` is older than years.
Your method passes in the `@published_date` attribute instead of `@last_played_at` attribute

Reply:
Hello, @skabeo I fixed it you can check the commit history. thanks for your feedback